src/ApiTestCase.php

```diff
@@ -59,7 +59,7 @@ protected function get($id)
 
     /**
      * @param Response $response
-     * @param int $statusCode
+     * @param int      $statusCode
```

Comment:
We do not align these anymore. I guess we need to tweak the php-cs-fixer configuration on Sylius etc. Task for later tho, please revert these.